scripts/python-skywater-pdk/skywater_pdk/fet_simulator/fet_simulator.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python3
```

Comment:
Maybe `skywater_pdk/simulation/fet.py` or maybe even `skywater_pdk/simulation/analog/fet.py`?

Reply:
I used the latter